tests/functional/test_utils.py

```diff
@@ -91,3 +143,14 @@ def file(*args):
 
     # empty file
     open(path, "a").close()
+
+    return path
+
+
+def read_link_without_junction_prefix(path: str):
+    # When our tests run on CI on Windows, it seems to use junctions, which causes symlink targets
+    # have a prefix. This function reads a symlink and returns the target without the prefix (if any).
+    target = os.readlink(path)
+    if target.startswith("\\\\?\\"):  # \\?\, with escaped slashes
+        target = target[4:]
+    return target
```

Comment:
duplicated from tests/functional/test_actions.py, since I needed to do the same. What's the best way to share this between tests, or should I just leave like this?

Reply:
maybe a testing_utils package?

Reply:
cool, might create a file for now

Reply:
done